content/en/docs/concepts/services-networking/service.md

```diff
@@ -859,6 +859,16 @@ these Services, and there is no load balancing or proxying done by the platform
 for them. How DNS is automatically configured depends on whether the Service has
 selectors defined:
```

Comment:
This ending should change as it refers to the selectors, but now it will lead to the headless service explanation.

Maybe move this last sentence to be it's own line between the last new line and the "With selectors" header.